tests/test_e2e.py

```diff
@@ -31,15 +33,34 @@ def test_end_to_end(numprocesses, tmp_path):
     """An end-to-end test to start the fuzzer and access the dashboard."""
     test_fname = tmp_path / "test_demo2.py"
     test_fname.write_text(TEST_CODE, encoding="utf-8")
-    dash_proc = entrypoint._fuzz_impl(
-        numprocesses=numprocesses,
-        dashboard=True,
-        port=7777,
-        unsafe=False,
-        pytest_args=["-p", "no:dash", str(test_fname)],
+    process = subprocess.Popen(
```

Comment:
I thought this might be better than trying to expose the dashboard process in the code. I think the alternative is to set `dash_proc` as a global somewhere, or maybe there's an easy way to return it that I'm not seeing.

Reply:
`subprocess` seems reasonable to me 👍